controllers/reaction_controller.py

```diff
@@ -1,10 +1,17 @@
 from discord import Message
 from db import DB
+from datetime import datetime, timedelta
 
 
 class ReactionController:
     @staticmethod
     async def apply_reactions(message: Message):
         emojis = DB().get_reactions_for_user(message.author.id)
-        for emoji in emojis:
-            await message.add_reaction(emoji)
+        if len(emojis) != 0: # prevents further DB calls if user does not have any Robomojis
```

Comment:
Would prefer an early return here rather than indenting this entire code block. It makes it a bit easier to read the control flow.

```python
if len(emojis) == 0:
    return
 emoji_delay_seconds = DB().get_emoji_reaction_delay()
# ...
```